components/date-picker/locale/de_DE.tsx

```diff
@@ -0,0 +1,8 @@
+import DatePickerLocale from "rmc-date-picker/lib/locale/en_US";
+
+export default {
+  okText: "OK",
+  dismissText: "Abrechen",
+  extra: "bitte auswählen",
```

Comment:
Should be capitalized.

```suggestion
  extra: "Bitte auswählen",
```